tools/webdriver/webdriver/servo.py

```diff
@@ -1,18 +0,0 @@
-class ServoExtensionCommands(object):
```

Comment:
If this were ever imported, it would throw `NameError` because `command` is undefined. Therefore, I concluded it was totally dead cold and could be removed.

Reply:
cc/ @jdm